test/connection/transaction_test.exs

```diff
@@ -0,0 +1,200 @@
+defmodule Janus.Connection.TransactionTest do
+  alias Janus.Connection.Transaction
+  import ExUnit.CaptureLog
+  import Mock
+  use ExUnit.Case
+
+  @tag :tag
+  @timeout 5000
+  @now %DateTime{
+    year: 2000,
+    month: 2,
+    day: 29,
+    zone_abbr: "CET",
+    hour: 23,
+    minute: 0,
+    second: 7,
+    utc_offset: 3600,
+    std_offset: 0,
+    time_zone: "Etc/UTC"
+  }
+
+  setup do
+    table = Transaction.init_transaction_call_table()
```

Comment:
This changes global state (ETS). The test has to be configured with `async: false` or you need to generate a unique name for this table

Reply:
Test module name as tab name?

Reply:
Whatever, just remember to clean up after the tests - especially if not using a unique table

Reply:
This doesn't seem to affect other tests suites because:
 - This used table is private, and only owner can read or write
 - It is not named, and it is registered by automatically generated reference returned by :ets.new

As discussed IRL, this table seems to dissapear before on_exit is called.

Reply:
ETS table dies when owner dies.